package.json

```diff
@@ -34,7 +34,7 @@
 		"prepare": "husky install",
 		"prebuild": "rimraf dist",
 		"dev": "npm run storybook",
-		"storybook": "start-storybook -p 6006 --no-manager-cache",
```

Comment:
Why is this flag being removed?

Reply:
From the docs

> The flag --no-manager-cache disables the internal caching of Storybook and can severely impact your Storybook loading time, so only use it when you need to refresh Storybook's UI, such as when editing themes.

I put it there sometime ago when I had problems with Storybook not updating but I think it wasn't even the solution, I forgot it there and committed. But there's no reason for this flag to be there.

Reply:
🔝 👍